clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/SeparatedStoreBufferService.java

```diff
@@ -22,11 +18,10 @@
  */
 public class SeparatedStoreBufferService extends AbstractStoreBufferService {
   private static final Logger LOGGER = LogManager.getLogger(SeparatedStoreBufferService.class);
-  protected final StoreBufferService sortedServiceDelegate;
-  protected final StoreBufferService unsortedServiceDelegate;
+  protected final StoreBufferService sortedStoreBufferServiceDelegate;
+  protected final StoreBufferService unsortedStoreBufferServiceDelegate;
```

Comment:
I think everything else looks good to me, but could you for the debugging purpose consider adding a new constructor with prefix and when creating writers use the prefix for this separated drainer case? Otherwise in the log it is hard to debug.
This should be just a simple constructor change for StoreBufferService and one extra variable for prefix default to ""

Reply:
I think this is a must have before enabling the feature again, otherwise, there is no way to operate.

Reply:
I think it is already addressed in the code. There is suffix in the writer thread.

Reply:
then that's great